app/controllers/v1/profiles_controller.rb

```diff
@@ -17,17 +17,20 @@ def index
       params[:sort_by] ||= 'score'
       render_json resolve_collection
     end
+    permission_for_action :index, Rbac::POLICY_READ
```

Comment:
This isn't correct, you should allow access to canonical policies for `COMPLIANCE_VIEWER` the rest is sorted out via Pundit.